template.yaml

```diff
@@ -176,7 +185,6 @@ Resources:
   ScopusEmailBucket:
     Type: AWS::S3::Bucket
     Properties:
-      AccessControl: Private
```

Comment:
This change is correct, but it may fail when you deploy the stack because CloudFormation is stupid. Let me know if it does, we can fix it manually.